components/control/Input/Input.css

```diff
@@ -20,6 +19,9 @@ diamond-input {
 
   &:focus-within {
     --_background: var(--diamond-input-background-focus);
+    /* https://css-tricks.com/copy-the-browsers-native-focus-styles/ */
```

Comment:
Should this be on :has(input:focus-visible)? I wouldn't expect the ring to show when I click on an input, just the border to change

Reply:
You always get a ring on inputs, that's the standard behaviour unless we override it.
I still think your idea is better though, will prevent conflicts with buttons in suffix